loader/cJSON.c

```diff
@@ -37,16 +37,21 @@
 #include "cJSON.h"
 
 #include "allocation.h"
+#include "loader.h"
+#include "log.h"
 
 void *cJSON_malloc(const VkAllocationCallbacks *pAllocator, size_t size) {
     return loader_alloc(pAllocator, size, VK_SYSTEM_ALLOCATION_SCOPE_COMMAND);
 }
 
 void cJSON_Free(const VkAllocationCallbacks *pAllocator, void *pMemory) { loader_free(pAllocator, pMemory); }
 
+/*
+// commented out as it is unused - static error code channel requires external locks to be used.
 static const char *ep;
 
 const char *cJSON_GetErrorPtr(void) { return ep; }
+*/
```

Comment:
So this is the json library you were talking to me about.

Is there value in leaving them as comments? IE why not just delete it entirely?